fossevents/events/services.py

```diff
@@ -51,6 +59,27 @@ def send_event_update_mail(event, moderators_email):
                extra_context, 'email/event_update_creator_email.html')
 
 
+def send_event_review_mail(review, moderator):
+
+    # get all moderator's email
+    moderators_email = User.objects.filter(Q(is_staff=True) | Q(is_moderator=True) |
+                                           Q(is_superuser=True)).distinct().values_list('email', flat=True)
```

Comment:
Filtering on `is_moderator` will fulfill the use-case here, `is_staff` & `is_superuser` not needed.